xmodule/tests/test_annotatable_block.py

```diff
@@ -119,6 +127,9 @@ def test_render_content(self):
         assert expected_num_annotations == actual_num_annotations, 'check number of annotations'
 
     def test_get_html(self):
+        if not hasattr(self.annotatable, "get_html"):
```

Comment:
Rather then skipping we should write test case for the similar functionality i-e django template rendering functionality which is replacing mako template functionality to keep the same test coverage.

Reply:
Agreed that we should not skip this. Is there a similar function that can be called on the extracted block instead of `get_html` which ensures that the right keys are present in the rendering context?